static/docs/command-reference/checkout.md

```diff
@@ -6,7 +6,7 @@ DVC-files.
 ## Synopsis
 
 ```usage
-usage: dvc checkout [-h] [-q | -v] [-d] [-f] [-R]
+usage: dvc checkout [-h] [-q | -v] [-d] [-R] [-f] [--relink]
```

Comment:
does it fit into a single line?

Reply:
Yep

![image](https://user-images.githubusercontent.com/1477535/71704075-c7988b80-2d9d-11ea-95e8-f99e73243fa8.png)
> https://dvc-landing-relink-ubdbmixgnby.herokuapp.com/doc/command-reference/checkout